packages/docs/src/pages/color-modes.mdx

```diff
@@ -30,9 +31,37 @@ for optional color modes.
 }
 ```
 
-The colors defined at the root of the `colors` object will be accessible as
-`default`. All colors found in `colors.modes` will be referenced by their key.
-The above example will have two modes: `default` and `dark`.
+All colors found in `colors.modes` will be referenced by their key in the
+context object `rawColors.modes`. The above example will have two modes: `light`
+and `dark`.
+
+## Initial colors
+
+The colors defined at the root of the `colors` are tranformed upon `colorMode`
+change. This is to allow for reference like `theme.colors.primary` to return:
+
+- `colors.primary`, when the color mode is set to its initial mode
+- `colors.modes.dark.primary`, when the color mode is set to `dark`
+
+Because of this, we store those initial colors with the other modes. They will
+be accessible as
+
+- `rawColors.modes.__default` (if `initialColorModeName` is undefined)
+- `rawColors.modes.light` (using the value of `initialColorModeName`).
+
+```js
+{
+  initialColorModeName: 'light',
+  rawColors: {
+    primary: '#07c',
+    modes: {
+      // __default: {}, initialColorModeName is defined
```

Comment:
>  // no '__default' here as initialColorModeName is defined

? or did I understand it wrong

Reply:
Nope, thats right! It was not super clear